lib/src/main/kotlin/com/swmansion/starknet/deployer/StandardDeployer.kt

```diff
@@ -0,0 +1,44 @@
+package com.swmansion.starknet.deployer
+
+import com.swmansion.starknet.account.Account
+import com.swmansion.starknet.data.selectorFromName
+import com.swmansion.starknet.data.types.Call
+import com.swmansion.starknet.data.types.Calldata
+import com.swmansion.starknet.data.types.Event
+import com.swmansion.starknet.data.types.Felt
+import com.swmansion.starknet.data.types.transactions.GatewayTransactionReceipt
+import com.swmansion.starknet.data.types.transactions.InvokeTransactionReceipt
+import com.swmansion.starknet.data.types.transactions.TransactionReceipt
+import com.swmansion.starknet.extensions.map
+import com.swmansion.starknet.provider.Provider
+import com.swmansion.starknet.provider.Request
+
+class StandardDeployer(
+    private val deployerAddress: Felt,
+    private val provider: Provider,
+    private val account: Account,
+) : Deployer {
+    override fun deployContract(classHash: Felt, salt: Felt, calldata: Calldata): Request<ContractDeployment> {
```

Comment:
We don't expose unique here. Why?

Reply:
It didn't seem useful exposing this parameter.

Reply:
Why is it not usefeul? It creates a virtual namespace based on your account by default.